config/ompi_setup_mpi_java.m4

```diff
@@ -0,0 +1,91 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+dnl                         University Research and Technology
+dnl                         Corporation.  All rights reserved.
+dnl Copyright (c) 2004-2006 The University of Tennessee and The University
+dnl                         of Tennessee Research Foundation.  All rights
+dnl                         reserved.
+dnl Copyright (c) 2004-2008 High Performance Computing Center Stuttgart,
+dnl                         University of Stuttgart.  All rights reserved.
+dnl Copyright (c) 2004-2006 The Regents of the University of California.
+dnl                         All rights reserved.
+dnl Copyright (c) 2006-2012 Los Alamos National Security, LLC.  All rights
+dnl                         reserved.
+dnl Copyright (c) 2007-2012 Oracle and/or its affiliates.  All rights reserved.
+dnl Copyright (c) 2008-2012 Cisco Systems, Inc.  All rights reserved.
+dnl Copyright (c) 2015      Research Organization for Information Science
+dnl                         and Technology (RIST). All rights reserved.
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+# This macro is necessary to get the title to be displayed first.  :-)
+AC_DEFUN([OMPI_SETUP_JAVA_BINDINGS_BANNER],[
+    opal_show_subtitle "Java MPI bindings"
+])
+
+# OMPI_SETUP_JAVA_BINDINGS()
+# ----------------
+# Do everything required to setup the Java MPI bindings.  Safe to AC_REQUIRE
+# this macro.
+AC_DEFUN([OMPI_SETUP_JAVA_BINDINGS],[
+    AC_REQUIRE([OMPI_SETUP_JAVA_BINDINGS_BANNER])
```

Comment:
any reason why you `AC_REQUIRE()` instead of simply inlining the (one line) macro ?

Reply:
I think this is a holdover -- there is a prior comment saying that it is required to get the title output in order.  But I think that's from prior days when OMPI_SETUP_JAVA_BINDINGS itself was AC_REQUIRED.  Doesn't seem like this is needed any more; I'll remove.